pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -748,3 +752,152 @@ func (os *OpenStack) GetZone() (cloudprovider.Zone, error) {
 func (os *OpenStack) Routes() (cloudprovider.Routes, bool) {
 	return nil, false
 }
+func (os *OpenStack) AttachDisk(diskName string) (string, error) {
```

Comment:
nit: add empty line between functions.
nit: Add godoc comments to public methods.